ban/commands/municipality.py

```diff
@@ -0,0 +1,147 @@
+from ban.commands import command, reporter
+from ban.core import models
+from ban.core import versioning
+from . import helpers
+
+
+@command
+@helpers.session
+def merge(destination, sources=[], name='', label='', **kwargs):
```

Comment:
That's the way we configure the command line arguments ;)
I'll show you that in more details during next pair session :)